config/src/main/java/org/hyperledger/besu/config/GenesisConfigOptions.java

```diff
@@ -71,8 +71,9 @@
 
   OptionalLong getLondonBlockNumber();
 
-  // TODO EIP-1559 change for the actual fork name when known
-  OptionalLong getAleutBlockNumber();
+  OptionalLong getArrowGlacierBlockNumber();
+
+  OptionalLong getBaseFeePerGas();
```

Comment:
Can all the BaseFeePerGas methods that are added be removed?  It's not needed for the bomb and not referenced anywhere else in this PR.

Reply:
yup.  can do.  This is one of the artifacts of the merge strategy for the cherry-pick.